kiwi/system/root_import/oci.py

```diff
@@ -100,6 +100,6 @@ def __del__(self):
             Path.wipe(self.oci_layout_dir)
         if self.oci_unpack_dir:
             Path.wipe(self.oci_unpack_dir)
-        if (self.uncompressed_image is not None and
-                self.uncompressed_image != self.image_file):
-            Path.wipe(self.uncompressed_image)
+        if (self.uncompressed_image):
```

Comment:
no complain about the extra parens?

Reply:
yeah that one was a leftover of the simplify of the condition, thanks